content/en/Getting started/Assets/asset-description.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Describe Your Assets"
+linkTitle: "Describe Your Assets"
+weight: 50
+description: >
+  Better descriptions help our pentesters test your assets properly.
+---
+
+{{% pageinfo %}}
+Help our pentesters test your assets more quickly.
+{{% /pageinfo %}}
+
+<!-- What I've set up is different from the "Asset Description" popup.
+In my judgment, the descriptions in our UI are not consistent. -->
+
+Our pentesters need all relevant information about your asset. To help
+you understand what to share, we include a description template.
+
+For all assets, we'd appreciate a:
+
+- High-level overview
+- Description of important functions or features
+- Business risks associated with each function and feature
+
+Include links to published documentation related to the
+asset. You can upload documentation, diagrams, and more in various
+file formats under [Asset Documentation](#asset-documentation).
+
+The following sections detail additional needs for different kinds of assets:
+
+### Web, API, Mobile
+
+Web, API, and Mobile assets frequently include user roles in different
+categories such as:
+
+- Administrator
+- Service user
+- Regular user
+
+Each of these roles typically have different sets of rights, privileges,
+or permissions. We can verify whether such roles are appropriately limited.
+
+For web assets, define the application type. For example, some web assets may be a:
+
+- Page-driven website
+- [Single-page application](https://developer.mozilla.org/en-US/docs/Glossary/SPA)
+
+### Web Asset Description
+
+Help us find the right pentesters for your asset. Include a high-level overview
+of the application. Add details such as:
+
+- Language, such as JavaScript or Ruby on Rails.
```

Comment:
I wonder if we should just leave this general, as "Coding languages" - given that almost all web apps have some flavor of javascript?

Reply:
Point well taken. "JavaScript" is sort of like "Kleenex" at this point.